frontend/src/pages/project/Overview.vue

```diff
@@ -62,43 +82,80 @@
             </div>
         </div>
     </div>
-
 </template>
 
 <script>
-import projectApi from '@/api/project'
-import { ExternalLinkIcon, TrendingUpIcon, TemplateIcon } from '@heroicons/vue/outline'
-import FormHeading from '@/components/FormHeading'
+import { ExternalLinkIcon, TemplateIcon, TrendingUpIcon } from '@heroicons/vue/outline'
+
+import { mapState } from 'vuex'
+
 import ProjectStatusBadge from './components/ProjectStatusBadge'
+
+import projectApi from '@/api/project'
 import AuditLog from '@/components/AuditLog'
+import DropdownMenu from '@/components/DropdownMenu'
+import FormHeading from '@/components/FormHeading'
+import SectionTopMenu from '@/components/SectionTopMenu'
+import permissionsMixin from '@/mixins/Permissions'
 
 export default {
     name: 'ProjectOverview',
-    props: ['project', 'isVisitingAdmin'],
+    components: {
+        AuditLog,
+        DropdownMenu,
+        ExternalLinkIcon,
+        FormHeading,
+        ProjectStatusBadge,
+        SectionTopMenu,
+        TemplateIcon,
+        TrendingUpIcon
+    },
+    mixins: [permissionsMixin],
+    props: {
+        project: {
+            required: true,
+            type: Object
+        },
+        isVisitingAdmin: {
+            required: true,
+            type: Boolean
+        }
+    },
+    emits: ['project-start', 'project-delete', 'project-suspend', 'project-restart'],
     computed: {
+        ...mapState('account', ['teamMembership']),
         options: function () {
-            return [
-                { name: 'Start', action: async () => { await projectApi.startProject(this.project.id) } },
-                { name: 'Restart', action: async () => { await projectApi.restartProject(this.project.id) } },
-                { name: 'Suspend', action: () => { this.$router.push({ path: `/project/${this.project.id}/settings/danger` }) } }
+            const flowActionsDisabled = !(this.project.meta && this.project.meta.state !== 'suspended')
+
+            const result = [
+                {
+                    name: 'Start',
+                    action: () => this.$emit('project-start'),
```

Comment:
I see this removes the explicit call to `projectApi.startProject(...)` in favour of emitting this event - but I don't see anything added to handle that event? Is that something that has already been added elsewhere?

Reply:
The project-start event bubbles up to the project index view:
 https://github.com/flowforge/flowforge/blob/514cb4dc87ee238f0b1c479e0847cff58d098815/frontend/src/pages/project/index.vue#L51

https://github.com/flowforge/flowforge/blob/514cb4dc87ee238f0b1c479e0847cff58d098815/frontend/src/pages/project/index.vue#L201-L204

Reply:
@Pezmc see comment above